SO_scrapper.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup as bs 
```

Comment:
The Stack Exchange family has an API which we can use here instead of scrapping.